src/debonair/Restaurant.java

```diff
@@ -0,0 +1,137 @@
+// user defined package
+package debonair;
+
+import java.time.LocalDateTime;
+// imports from in-built packages
+import java.util.Iterator;
+import java.util.Scanner;
+
+// importing classes from other packages
+import resources.Constants;
+
+// Restaurant class
+public class Restaurant implements IRestaurant {
+	/**
+	 * This method is used to show menu to the customer
+	 */
+	public void showMenu() {
+		System.out.println("The menu has following items:-");
+		// Iterator created to print items from Menu class's itemMenu List
+		Iterator<Item> itemsIterator = Menu.ITEM_MENU.iterator();
+		while (itemsIterator.hasNext()) {
+			Item item = itemsIterator.next();
+			System.out.println(item.toString());
```

Comment:
No need to call toString.Code can be clubbed into single line to remove variable decleration
```suggestion
			System.out.println(itemsIterator.next());
```